buildSrc/src/main/java/Dependencies.kt

```diff
@@ -96,6 +97,7 @@ object Deps {
     const val tools_espresso_core = "com.android.support.test.espresso:espresso-core:${Versions.tools_espresso_core}"
     const val tools_espresso_idling_resources = "com.android.support.test.espresso:espresso-idling-resource:${Versions.tools_espresso_version}"
     const val tools_espresso_web = "com.android.support.test.espresso:espresso-web:${Versions.tools_espresso_version}"
+    const val tools_test_okhttp3 = "com.squareup.okhttp3:mockwebserver:${Versions.tools_test_okhttp3}"
```

Comment:
Let's follow the naming in AC and name this "mockwebserver" - since it's not the "okhttp3" library itself that we are importing here.